quiz.js

```diff
@@ -0,0 +1,25 @@
+"use strict"
+
+function populatePage(inventory) {
+  // Loop over the inventory and populate the page
+    let outputString = "";
+    let output=document.getElementById("output");
+  for (var i = 0; i < inventory.length; i++) {
+    let currentCar = inventory[i];
+    outputString +=`<div class="col-xs-4"><div id="eachCar${i}" class="car"><p>${currentCar.year}  ${currentCar.make}  ${currentCar.model}</p><p>Price:  ${currentCar.price}</p>  Color:  ${currentCar.color}  <p>${currentCar.purchased}</p><p id="carDescription${i}">Description:  ${currentCar.description}</p></div></div>`
+
+      output.innerHTML = outputString;
+      };
+  if (i<=3){
+          document.getElementById("eachCar0").classList.add("fireBrick");
+          document.getElementById("eachCar1").classList.add("black");
+          document.getElementById('eachCar2').classList.add("blue");
```

Comment:
This is an interesting solution to the problem at hand, but food for thought: What happens if the JSON data changes and you have different cars with different colors? What happens when an additional car is added to the inventory? This limits you to exactly three cars with exactly these three colors. Something to think about when you build something else that pulls in data from an outside source and you don't know what you're getting back from the XHR call.